APMrover2/AP_MotorsUGV.h

```diff
@@ -98,7 +98,7 @@ class AP_MotorsUGV {
     AP_Int8 _pwm_type;  // PWM output type
     AP_Int8 _pwm_freq;  // PWM output freq for brushed motors
     AP_Int8 _disarm_disable_pwm;    // disable PWM output while disarmed
-    AP_Int8 _slew_rate; // slew rate expressed as a percentage / second
+    AP_Int16 _slew_rate; // slew rate expressed as a percentage / second
```

Comment:
We need to be careful about changing the type of the parameter because it leads to parameter corruption for any users that have set that parameter to a non-default value.  Because this is a new parameter for Rover-3.2 (the whole AR_MotorsUGV class is new) this issue would only affect beta testers.   Still, I think to avoid this problem we should change the eeprom location of the parameter in the .cpp file (https://github.com/ArduPilot/ardupilot/blob/master/APMrover2/AP_MotorsUGV.cpp#L55)... just change it from "4" to "8".

By the way, if this was not a new parameter for Rover-3.2.0, we could copy over the old value to the  new location by adding a line to the "conversion_table" array.  https://github.com/ArduPilot/ardupilot/blob/master/APMrover2/Parameters.cpp#L537.  Plane makes the most extensive use of this kind of parameter copying so it's table has the most good examples of this.  https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/Parameters.cpp#L1212